src/test/java/uk/gov/homeoffice/digital/sas/balancecalculator/BalanceCalculatorDeleteActionIntegrationTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.homeoffice.digital.sas.balancecalculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.homeoffice.digital.sas.balancecalculator.testutils.CommonUtils.createTimeEntry;
+
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual.Accrual;
+import uk.gov.homeoffice.digital.sas.balancecalculator.models.timecard.TimeEntry;
+import uk.gov.homeoffice.digital.sas.kafka.message.KafkaAction;
+
+@SpringBootTest
+@AutoConfigureWireMock(port = 9999)
+class BalanceCalculatorDeleteActionIntegrationTest {
+
+  private static final String TIME_ENTRY_ID = "a90a037a-a6c5-4fc0-9e72-a533c8c89c68";
+  private static final String TENANT_ID = "52a8188b-d41e-6768-19e9-09938016342f";
+  private static final String PERSON_ID = "b810a175-9b87-473c-94f3-9bcdf87f3a58";
+
+  @Autowired
+  private  BalanceCalculator balanceCalculator;
+
+  @Test
+  void calculate_endToEnd_contributionsAndCumulativeTotalsAsExpected() {
```

Comment:
the second clause in the method name could be more descriptive. Is this dealing with the BST to GMT switch? If so the second clause could be renamed to something along the lines of:  _bstToGmtSwitch_

Reply:
Tests updated to avoid BST to GMT and test names updated to be more descriptive.